package.json

```diff
@@ -36,14 +36,16 @@
     "sass-loader": "4.0.0",
     "style-loader": "0.13.1",
     "webpack": "1.13.1",
-    "webpack-dev-server": "1.14.1"
+    "webpack-dev-server": "1.14.1",
+    "react-dev-utils": "^0.5.0"
   },
   "dependencies": {
     "markdown-loader": "^2.0.0",
     "marked": "^0.3.6",
     "react": "15.2.1",
     "react-dom": "15.2.1",
     "semantic-ui-react": "^0.67.0",
+    "semantic-ui-sass": "^2.1.7",
```

Comment:
Semantic-UI-Sass port, sadly they seem blocked on going 100% that way (sadly their progress on the matter is a bit blocked) https://github.com/Semantic-Org/Semantic-UI/issues/1147